core/include/misc/uri.h

```diff
@@ -0,0 +1,50 @@
+//
+// Created by Jake Bolewski on 7/20/17.
+//
+
+#ifndef TILEDB_URI_H
+#define TILEDB_URI_H
+
+#include <string>
+#include "status.h"
+
+namespace tiledb {
+
+namespace uri {
```

Comment:
if you have functions, it is nice to namespace them so that not everything has to be associated with a class